nav2_bt_navigator/src/bt_navigator.cpp

```diff
@@ -72,6 +72,12 @@ BtNavigator::BtNavigator()
   declare_parameter("global_frame", std::string("map"));
   declare_parameter("robot_base_frame", std::string("base_link"));
   declare_parameter("odom_topic", std::string("odom"));
+  declare_parameter("pose_enable_groot_monitoring", true);
+  declare_parameter("pose_groot_zmq_publisher_port", 1670);
+  declare_parameter("pose_groot_zmq_server_port", 1671);
+  declare_parameter("poses_enable_groot_monitoring", true);
```

Comment:
I think the `enable_groot_monitoring` should remain and apply to both `pose` and `poses`. I think that should be an "all or nothing" parameter.

Reply:
Thanks for the comments! I updated the PR with your suggestions. However, I seem to have problem connecting to one of the groot servers, I'll see if I can have a look at it either over the weekend or early next week.